contrib/blueflood-docker/configs/ES-Setup/index_settings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
The original one is located at ./blueflood-elasticsearch/src/main/resources/index_settings.json